src/front/shared/redux/reducers/oneinch.ts

```diff
@@ -41,7 +41,7 @@ interface State {
 }
 
 const blockchains = {}
-const allowedCurrency = ['ETH', 'BNB', 'MATIC', 'XDAI', 'AVAX']
+const allowedCurrency = ['ETH', 'BNB', 'CNDL', 'MATIC', 'XDAI', 'AVAX']
```

Comment:
I made changes, you don't need to put it here.

Reply:
Should I remove it?